torchmultimodal/transforms/text_transforms.py

```diff
@@ -30,7 +30,7 @@ def __call__(
         self, l: Union[List[str], List[List[str]]]
     ) -> Union[List[int], List[List[int]]]:
         if isinstance(l[0], str):
-            return [int(x) for x in l]
+            return [int(x) for x in l]  # type: ignore
```

Comment:
why ignore for only this one?

Reply:
There's an assumption here that every element of l is either str or List[str]. So we only check that the first one is a str. But then mypy complains because it thinks the other elements could still be List[str]. Personally I would rather ignore the error than iterate over the whole list just to make mypy happy